test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -1173,20 +1354,22 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 		BeforeEach(func() {
 			//Saving the old performance profile
 			initialProfile = profile.DeepCopy()
-		})
 
-		It("[disruptive] should set offline cpus after deploy PAO", func() {
 			for _, node := range workerRTNodes {
 				onlineCPUCount, err := nodes.ExecCommandOnNode([]string{"nproc", "--all"}, &node)
 				Expect(err).ToNot(HaveOccurred())
+
 				onlineCPUInt, err := strconv.Atoi(onlineCPUCount)
 				Expect(err).ToNot(HaveOccurred())
+
 				Expect(onlineCPUInt).Should(BeNumerically(">=", 3))
 				if onlineCPUInt <= 8 {
-					Skip(fmt.Sprintf("This test needs more than 8 CPUs online to work correctly, current online CPUs are %s", onlineCPUCount))
+					Skip(fmt.Sprintf("Offlined CPU API tests need more than 8 CPUs online to work correctly, current online CPUs are %s", onlineCPUCount))
```

Comment:
you are still ignoring the after matching after each section

Reply:
It is not possible to check and skip in the Context section. The best we can do is here right at the beginning